tutorials/observing_examples.ipynb

```diff
@@ -0,0 +1,587 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "T_eff = 4780  # temperature of hatp11[K], Bakos et al. 2010\n",
+    "\n",
+    "# Used the phoenix module in aesop.py to generate a model spectrum for T_eff = 4780\n",
+    "photlam_hatp11 = SourceSpectrum.from_file('phoenix_model_spectrum.dat', flux_unit = u.erg / u.s / u.cm ** 2 / u.cm)\n",
```

Comment:
Let's do some astropy magic here rather than including a data file in the PR. You can [construct a `SourceSpectrum` from a wavelength and flux array like so](https://synphot.readthedocs.io/en/latest/synphot/spectrum.html#arrays), and you can use astropy's FITS/downloading machinery like I did [in this notebook](https://gist.github.com/bmorris3/d661dd739c5c0f44f61daf2b9cca1d2a) in order to get the wavelength and flux arrays.